cmd/argo/commands/resume.go

```diff
@@ -18,15 +29,22 @@ func NewResumeCommand() *cobra.Command {
 				cmd.HelpFunc()(cmd, args)
 				os.Exit(1)
 			}
+			parsedStepSelector, err := fields.ParseSelector(resumeArgs.stepSelector)
+			if err != nil {
+				log.Fatal(err)
+			}
+
 			InitWorkflowClient()
 			for _, wfName := range args {
-				err := util.ResumeWorkflow(wfClient, wfName)
+				err := util.ResumeWorkflow(wfClient, wfName, resumeArgs.stepFailMessage, parsedStepSelector)
 				if err != nil {
 					log.Fatalf("Failed to resume %s: %+v", wfName, err)
 				}
 				fmt.Printf("workflow %s resumed\n", wfName)
 			}
 		},
 	}
+	command.Flags().StringVar(&resumeArgs.stepFailMessage, "stepFailMessage", "", "Fail suspended step with a message")
+	command.Flags().StringVar(&resumeArgs.stepSelector, "stepSelector", "", "Resume step parameter selector")
```

Comment:
These should have CLI-style casing. E.g. `step-fail-message` and `step-selector`

Reply:
👍